### DISCUSSION TITLE
How to surface light effect and speed in an HA card?

### DISCUSSION THREAD

**Discussion by soupman98** [cryptk/haomnilogic-local#121] [Q&A] [answered]
I can see it, but can't surface it elsewhere... and would like to 
![Pool light hayward issue](https://github.com/user-attachments/assets/39a9ea27-ac1c-4c2e-95cc-241f951a8a9a)

**Comment by cryptk** [comment, marked as answer]
Sorry I'm just now seeing this. I'm not sure what you mean by "surface it elsewhere", but you should be able to use the built-in entity card and then click/tap on the light in the card to view the more-info dialog to control it.

Other options are the excellent [Mushroom cards](https://github.com/piitaya/lovelace-mushroom) available in HACS or the [Custom Light Entity Card](https://github.com/ljmerza/light-entity-card) if you want something that can put the effect selector right on the dashboard without requiring an extra click.

Either way, this isn't really about the integration, but rather Home Assistant dashboard creation, and I would recommend either the Home Assistant Discord or the Community Forums for further help with dashboard design.
